README.md

```diff
@@ -135,7 +135,7 @@ public class MyService
     public object Post(Poco request)
     {
         var response = new Poco { Id = 2, Name = "New Poco from request" };
-        using (var db = HostContext.AppHost.Container.Resolve<IDbConnectionFactoryExtended>().Open())
+        using (var db = HostContext.AppHost.Container.Resolve<IDbConnectionFactory>().Open())
         {
             db.Insert(response);
         }
```

Comment:
ServiceStack Services has a built-in `base.Db` property most users would use instead. Outside of ServiceStack you can use the more succinct `HostContext.Resolve<IDbConnectionFactory>()` API.